### PR TITLE
Covid-19 Tracker

### PR DESCRIPTION
**Pull request by usorfaitheloho**
# React Capstone Project

## In this project, I followed the project requirement which are

- Project requirements

- [Narrativa API](https://covid19tracking.narrativa.com/index_en.html): COVID-19 data.
.
You can find many APIs in [this GitHub repo](https://github.com/public-apis/public-apis) or in [ProgrammableWeb](https://www.programmableweb.com/category/all/apis). Some of the APIs requires a token for authentication and some others are just open.

- Design

You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.
Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).

### Links
- you can watch my video presentation here[this video](https://www.loom.com/share/fc770c3bd8cf4543b1c050ab5e52aa0b)
-  This is a link to my live version [live link](https://covid-19-data-app.netlify.app/)

Live Demo
[Covid Tracker Live Demo](https://covid-19-data-app.netlify.app/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rustam-tolipov) [inline README.md:71]: - [ ] [OPTIONAL] It seems that you forgot to add an MIT license to your project. Kindly, add the license to your project and personalize it.

File: src/tests/services.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import {
+  updateCountryNames,
+
+} from '../services';
+
+describe('covid19 tracker services', () => {
```
Comment (rustam-tolipov) [inline src/tests/services.test.js:6]: - [ ] Perfectly, you wrote a test for your project! I would suggest you write a little more about it. The main one is, your tests are failing you can see the screenshot:

Screen Shot 2022-04-22 at 10 52 29 PM

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello TEAM, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @usorfaitheloho, 

Great job implementing all the changes requested by your previous reviewer 😎
Your project is complete!  There is nothing else to say other than... it's time to merge it 😁
### Congratulations 🙌

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (usorfaitheloho) [reply README.md:71]: Thanks for the suggestion.
I have implemented the changes

File: src/tests/services.test.js (hunk 2 above)
Comment (usorfaitheloho) [reply src/tests/services.test.js:6]: Thanks for this suggestion.I have added more tests and fixed the error

**Comment by usorfaitheloho** [comment]
Hello @mateo951 ,

Thanks for approving my project 😸

**Pull request merged**